### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by luzpaz**
Various typos

### CODE REVIEW

**Review by marcfehling — APPROVED** [review]

**Comment by marcfehling** [comment]
Our CI is complaining about an indentation problem. Please run [`make indent`](https://github.com/dealii/dealii/wiki/Indentation) to fix the problem. If you end up with multiple commit, you can squash them into one.

Thank you for fixing the typos! It appears you went through a fairly random set of documentation, which leads me to a follow-up question: did you use an automated tool? If so, would you mind sharing with us which one you used? It would be great to add a typo check to our automated tests!

**Comment by luzpaz** [comment]
```shell
$ make indent
make: *** No rule to make target 'indent'.  Stop.
```

How do I run it ?

**Review by bangerth — APPROVED** [review]
Very cool -- thank you for the effort!

File: examples/step-60/step-60.cc [hunk 1]
```diff
@@ -299,7 +299,7 @@ namespace Step60
     void setup_embedded_dofs();
 
     // The only unconventional function we have here is the `setup_coupling()`
-    // method, used to generate the sparsity patter for the coupling matrix $C$.
+    // method, used to generate the sparsity pattern for the coupling matrix $C$.
```
Comment (marcfehling) [inline examples/step-60/step-60.cc:302]:
```suggestion
    // method, used to generate the sparsity pattern for the coupling matrix
    // $C$.
```

**Review by marcfehling — COMMENTED** [review]
You need to run it from your build directory. You will also need to install `clang-format` for this purpose.

On the other hand, I just checked which lines the CI is [actually reporting](https://github.com/dealii/dealii/actions/runs/3588121277/jobs/6039163349), and it's really just one line you need to fix. It's up to you if you pick `make indent` or fix the line directly.

**Review by tamiko — APPROVED** [review]

**Comment by luzpaz** [comment]
@marcfehling i used the (latest dev version of) codespell and some deeper regex searches with vscode to root our some typos.

**Pull request merged**